04-onion-routing.md

```diff
@@ -470,6 +470,15 @@ func NewOnionPacket(paymentPath []*btcec.PublicKey, sessionKey *btcec.PrivateKey
 	var mixHeader [routingInfoSize]byte
 	var nextHmac [hmacSize]byte
 
+        // Our starting packet needs to be padded with random bytes, so we read
+        // some here.
+        firstHopSize := hopData[numHops-1].NumBytes()
+	randBytes := make([]byte, routingInfoSize-firstHopSize)
+	if _, err := rand.Read(randBytes[:]); err != nil {
```

Comment:
can be made simpler/more efficient: https://github.com/lightningnetwork/lightning-onion/pull/40#discussion_r342891351

Reply:
Implemented, just to actually fill out the entire set of bytes.